src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -221,7 +221,7 @@ async def _aadd_embeddings(
         **kwargs: Any,
     ) -> List[str]:
         if not ids:
-            ids = ["NULL" for _ in texts]
+            ids = [str(i) for i, _ in enumerate(texts)]
```

Comment:
We need to generate an uuid:
```suggestion
            ids = [str(uuid.uuid4()) for _ in texts]
```